plasmapy/particles/particle_class.py

```diff
@@ -204,7 +205,32 @@ def json_dumps(self, **kwargs) -> str:
         return json.dumps(self.json_dict, **kwargs)
 
 
-class Particle(AbstractParticle):
+class AbstractPhysicalParticle(AbstractParticle):
+    """Base class for particles that are defined with physical units."""
+
+    @property
+    def _as_particle_list(self):
+        from plasmapy.particles.particle_collections import ParticleList
```

Comment:
The import statement is here to avoid circular imports.

Reply:
Figured!

Reply:
Probably better to put that in as a comment, rather than here! ;0